README.md

```diff
@@ -109,5 +102,14 @@ Looking through the `monorepo` we can expect the following workspaces:
 - `todo`
 - `vault`
 - `xcmetrics`
+</details>
 
-Of course, there could be simplifcations to this workspace list and some workspaces will be merged into one, like for example the `github-` workspaces could become one `github` workspace instead, but for the inital migration, we will keep them separate.
+### Migration process
+
+The migration of plugins from the `backstage/backstage` monorepo to the `community-plugins` repository was automated under the `community-cli` tool.
+
+You provide it with a path to the `monorepo` which should be cloned locally, and a plugin ID. It will then create a new workspace in the `community-plugins` repository with all of the plugins and modules that surround that workspace. For instance, if I use the `todo` plugin as an ID, It will automatically move over `@backstage/plugin-todo` as well as `@backstage/plugin-todo-backend` and any other `-common`, `-node` or `-modules` that are related.
```

Comment:
Hey! 👋 

Looks good! I am just wondering, reading this as someone who has not closely followed this process (apologies for any silly remarks!) and I got a bit confused here, is this meant to be as a "If you'd like to migrate your plugin to this repo then do the following...."?

Reply:
> No I do not believe so - as the automation was/is very specific to migrating from `backstage/backstage`. I think it could potentially be removed.

Ah clear, sounds like it could be useful but maybe not in the main README of the repo? 🤔 💭 Not sure how others feel about it 😅 

> Incidentally, I started to write a guide on migrating plugins from other locations last night, I'll open a draft for it

Awesome! This would be super! 👍

Reply:
Take this with a pinch of salt as it's not tested, but I don't think the requirement for it to be `backstage/backstage` is a hard one. `--monorepo-path` just needs to be a `yarn workspace`, so you should be able to point it at any repo that's checked out locally and it do the job for you, **but** it will try to create changesets and stuff which might not be necessary in the parent repo.

Could be worth trying this see if it works for end users (👀 @Parsifal-M) and expanding the CLI a little bit with options to drop some steps if they're not required in the parent repo. The CLI does a LOT of stuff for you which would be a shame to have to have as manual steps :pray:

Reply:
> Take this with a pinch of salt as it's not tested, but I don't think the requirement for it to be `backstage/backstage` is a hard one. `--monorepo-path` just needs to be a `yarn workspace`, so you should be able to point it at any repo that's checked out locally and it do the job for you, **but** it will try to create changesets and stuff which might not be necessary in the parent repo.
> 
> Could be worth trying this see if it works for end users (👀 @Parsifal-M) and expanding the CLI a little bit with options to drop some steps if they're not required in the parent repo. The CLI does a LOT of stuff for you which would be a shame to have to have as manual steps 🙏

Hmm interesting! I could try it out and see what happens :sweat_smile: I wll see if I get some time next week or so!